sdk/core/Azure.Core/src/ETagConverter.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure
+{
+    internal class ETagConverter: JsonConverter<ETag>
```

Comment:
Should we also have a converter for "ETag?", i.e. nullable ETag?

Reply:
Serializer implementations already support this and this PR has tests to verify.

Reply:
My understanding is that the default implementation will serialize null ETag to JSON null value. Is that the behavior we want? If an input type has a nullable ETag property, do we want a null value to serialize to JSON null, or we want to simply not serialize this property. Is there even a way to control this behavior at ETag level?

Reply:
There are serializer specific ways to control this. For example https://docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonignorecondition?view=net-5.0 or https://www.newtonsoft.com/json/help/html/JsonPropertyPropertyLevelSetting.htm